terraform/app.tf

```diff
@@ -18,8 +18,9 @@ resource "cloudfoundry_app" "beis-roda-app" {
     "RAILS_ENV"                              = "production"
     "DATA_MIGRATE"                           = var.data_migrate
     "SECRET_KEY_BASE"                        = var.secret_key_base
-    "DOMAIN"                                 = var.domain
-    "ADDITIONAL_DOMAINS"                     = var.additional_domains
+    "CANONICAL_HOSTNAME"                     = var.canonical_hostname
+    "DOMAIN"                                 = "https://${var.canonical_hostname}"
+    "ADDITIONAL_HOSTNAMES"                   = var.additional_hostnames
```

Comment:
Yeah, maybe `DOMAIN` is not the right naming here. Something like `ROOT_URI` would be more sensible I think.

Reply:
`DOMAIN` is a more specific term than `URI` -- for example, this cannot be any URL, it can only be a scheme and host. For example it could contain `https://example.com` but not `https://example.com/page.html`. Appropriate terms for this are "domain" or "origin".

If this variable can actually contain arbitrary URLs, then calling it a URL or URI would be fine.

Reply:
("domain" is a badly defined word but enough people use it to mean either "hostname" or "origin" or "site" (in the browser security model sense) that it's probably fine.)

Reply:
No problem - I am happy!